src/typings/contract.d.ts

```diff
@@ -0,0 +1,6 @@
+import { ABIDefinition } from "web3/eth/abi";
+
+export interface Contract {
```

Comment:
Cann we call this `SmartContract` ? I think it's important to get those symbols as generic but specific at the same time as we can..